src/Jupyter/Visualization/MeasurementHistogramContent.cs

```diff
@@ -0,0 +1,28 @@
+#nullable enable
```

Comment:
We'll need to make sure that this file has the same copyright header as other C# files in the repository.